lib/avro_turf/disk_cache.rb

```diff
@@ -83,6 +87,20 @@ def store_by_version(subject, version, schema)
     @schemas_by_subject_version[key]
   end
 
+  # Parse the file from disk, if it exists and is not zero length
+  private def read_from_disk_cache(path)
+    hash = nil
+    if File.exist?(path)
+      if File.size(path)!=0
+        hash = JSON.parse(File.read(path))
```

Comment:
You can just `return` here and avoid the `hash` variable. For extra clarity, you can instead `return nil` at the end of the method.

Reply:
Done